ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -53,6 +53,7 @@ FCU:
   - return_air_relative_humidity_sensor
   - discharge_air_temperature_sensor
   - failed_communication_alarm
+  - air_pressure_sensor
```

Comment:
This does not belong on the `FCU` type. As you've noted you are trying to use air pressure to determine if the fan is running. Please use the existing field `discharge_air_static_pressure_sensor` (which is already included on the `DFSS` type).

Reply:
noted ma'am. will remove this and the run_status above this message. (somehow it lost commenting feature)